slp_abacus/resources/default-mapping/default-mapping-file.yaml

```diff
@@ -0,0 +1,20 @@
+trustzones:
```

Comment:
Same as the Abacus source file, should be moved to the examples/abacus folder under the repo root. In this case, it would be better to rename it to something like `iriusrisk-abacus-mapping.yaml` to be more coherent with the rest of the mappings for other formats.